src/components/ViewDeck/ViewDeck.js

```diff
@@ -1,94 +1,99 @@
-import React, { Component } from "react";
+import React, { useState, useEffect } from "react";
 import { connect } from "react-redux";
-import mapStoreToProps from "../../redux/mapStoreToProps";
+// mapStoreToProps vs. mapStateToProps?
+// import mapStoreToProps from "../../redux/mapStoreToProps";
```

Comment:
mapStoreToProps is incorrect naming. Think of it this way:
Store = A Redux toolbox that holds all the tools needed to house and manipulate state.
State = the information you have in the store.
So you aren't ever passing the store to a component. Only state.

Reply:
I need to re-read the docs for mapStateToProps.  I'm not sure what my disconnect with this concept is but i'm having a really difficult time understanding the significance of importing mapStateToProps when we're using the const mapStateToProps at the end of the component to get our data from the redux store.  I'm so sorry you've explained this to me multiple times, I'm not trying to be thick skulled about it.